### PR TITLE
feat(your-score): [WIP] YourScore

### PR DESCRIPTION
**Pull request by ChefHutch**
- Branch for 'your score' Trading Competition component

[ ] Before opening a pull request, please read the [contributing guidelines](https://github.com/pancakeswap/pancake-frontend/blob/master/CONTRIBUTING.md) first
[ ] If your PR is work in progress, open it as `draft`
[ ] Before requesting a review, all the checks need to pass
[ ] Explain what your PR does

### CODE REVIEW

File: src/views/TradingCompetition/components/YourScore/CardUserInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    }
+    if (!registered) {
+      return 'You’re not participating this time.'
+    }
+    if (profile) {
+      return `@${profile.username}`
+    }
+    return ''
+  }
+
```
Comment (hachiojidev) [inline src/views/TradingCompetition/components/YourScore/CardUserInfo.tsx]: Gentle reminder to translate these.

File: src/views/TradingCompetition/components/YourScore/CardUserInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  const getSubHeadingText = () => {
+    if (!account) {
+      return 'Connect wallet to view'
+    }
+    if (!registered) {
+      return 'Sorry, you needed to register during the “entry” period!'
+    }
+    if (profile) {
+      return `${profile.team.name}`
```
Comment (hachiojidev) [inline src/views/TradingCompetition/components/YourScore/CardUserInfo.tsx]: Same here

File: src/views/TradingCompetition/components/YourScore/ScoreProfile.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`
+
+const LaurelWrapper = styled.div<{ dir?: 'l' | 'r' }>`
+  transform: ${({ dir }) => (dir === 'l' ? 'rotate(30deg)' : 'rotate(-30deg)')};
```
Comment (hachiojidev) [inline src/views/TradingCompetition/components/YourScore/ScoreProfile.tsx:19]: Please use `left` and `right`

Comment (ChefHutch) [reply src/views/TradingCompetition/components/YourScore/ScoreProfile.tsx:19]: Oh this shit is getting ripped out now we have Laurels in the ui kit ✊

File: src/views/TradingCompetition/components/YourScore/CardUserInfo.tsx (hunk 2 above)
Comment (ChefHutch) [reply src/views/TradingCompetition/components/YourScore/CardUserInfo.tsx]: Will do man - picking this work up again now

**Comment by ChefHutch** [comment]
Work being continued in this PR https://github.com/pancakeswap/pancake-frontend/pull/793

**Pull request closed without merging**